src/modules/Dropdown/Dropdown.js

```diff
@@ -686,10 +702,12 @@ export default class Dropdown extends Component {
       search && searchQuery && 'filtered'
     )
     let _text = placeholder
-    if (text && !searchQuery) {
-      _text = text
-    } else if (searchQuery) {
+    if (searchQuery) {
       _text = null
+    } else if (text) {
+      _text = text
+    } else if (open && !multiple) {
+      _text = this.getSelectedItemText()
```

Comment:
I see, let's update the naming then.  The SUI `selected` class means it is highlighted, so all code usage of the term also means highlighted.  The class SUI `active` class means it the item is part of the Dropdown value.

So, a user "selects" items by arrow up/down, then they make one "active" by click, tab, or enter.